doc/recipe/preprocessor.rst

```diff
@@ -1084,6 +1084,11 @@ Parameters:
     regions in the shape file is masked out. If ``true``, the regions in the
     shapefiles are masked out separately, generating an auxiliary dimension
     for the cube for this.
+  * ``ids``: by default, ``[]``, in this case all the shapes in the file will
+    be used. If a list of IDs is provided, only the shapes matching them will
+    be used. The IDs are assigned from the ``name`` or ``id`` attributes (in
+    that order of priority) if present in the file or from the reading order
```

Comment:
I understand from the code that it assigns numerical id's in reading order if the `name` and `id` attributes are not present. This could potentially result in unwanted and unpredictable behavior so it should be very clear in the documentation. Maybe an example that shows how this works might help. Currently it is only covered in a couple of words which are not completely clear.

Reply:
Updated